change/@ni-nimble-angular-2e46c39b-28a8-4aef-9b90-ecc3ef77f46e.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Some questions for my own understanding:
1. How does this impact clients? Does it change how we'd recommend they do imports? Or change what can be tree shaken?
2. If it does change anything:
   1. are there any docs or examples that need updating?
   2. are there any clients we should update ourselves or broadcasting we should do to get clients to change their code?
3. How does this relate to our vision of having separate secondary entry points for each component? (Issue #172 with some initial implementation in #1252 and #1336). Does this make that easier? Harder? Unrelated?

Reply:
1. No impact, what is exported from public-api.ts doesn't change. Doesn't help or hurt tree shaking. Tree-shaking is broken until we have secondary entry points.
2. N/A
3. Doesn't really help or hurt the future goal of secondary entry points. It's only for dev purposes.
   
   I'm on the fence of whether this change actually improves dev experience as-is. We have to deal with some complexities to avoid exporting symbols twice which is captured in the top-level public-api.ts. I think at worst this change could obfuscate those considerations and make them harder to understand / debug and at best is neutral for dev efficiency.

I'd probably advocate for not making this change as I don't think it gives a significant benefit for clients or nimble devs. Instead we should prefer secondary entry points as that will have tangible benefits for both in the future.

Reply:
I am not personally invested in this change, and I think it's marginally better to avoid any risk, so I'm going to abandon.